lib/modules/session/index.js

```diff
@@ -441,6 +441,18 @@ mage.core.cmd.registerMessageHook('mage.session', false, function (state, cfg, b
 
 
 mage.core.msgServer.msgStream.addAddressType('session', {
+	checkInterval: sessionTTL / 2 * 1000,
+	check: function (address, host, cb) {
+		try {
+			const fullSessionKey =	address.split('/')[1];
```

Comment:
I think the only way parsing can fail is if `address` is not a string. In other words, maybe a simple `typeof` check would be more appropriate than this try/catch construction with an asynchronous function call inside.

Reply:
`try/catch` is what is currently being used elsewhere (see line 457 onward).

Regarding your second suggestion, will do.